xroad-catalog-lister/src/main/resources/lister-production.properties

```diff
@@ -1,24 +1,15 @@
 logging.level.org.springframework.web=DEBUG
 logging.level.org.springframework.ws=DEBUG
 logging.level.org.hibernate=ERROR
+# Hibernate log configuration
+logging.level.org.hibernate.SQL=TRACE
```

Comment:
Asking just in case - is it okay to leave these TRACE logging levels in, in a file that has 'production' in its name? :)

Reply:
Nice catch. Removed them